rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/core/csv/CourseOfStudyMapper.java

```diff
@@ -0,0 +1,96 @@
+package org.niewidoczniakademicy.rezerwacje.core.csv;
+
+import lombok.AllArgsConstructor;
+import org.niewidoczniakademicy.rezerwacje.core.model.csv.DatabaseException;
+import org.niewidoczniakademicy.rezerwacje.core.model.database.CourseOfStudy;
+import org.niewidoczniakademicy.rezerwacje.core.model.database.Faculty;
+import org.niewidoczniakademicy.rezerwacje.core.model.database.SystemUser;
+import org.niewidoczniakademicy.rezerwacje.dao.repository.FacultyRepository;
+import org.niewidoczniakademicy.rezerwacje.dao.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Service
+@AllArgsConstructor(onConstructor = @__(@Autowired))
+public class CourseOfStudyMapper {
+
+    private final UserRepository userRepository;
+    private final FacultyRepository facultyRepository;
+
+    public CourseOfStudy convert(org.niewidoczniakademicy.rezerwacje.core.model.csv.CourseOfStudy cos) {
+        return CourseOfStudy.builder().build();
+    }
+
+    public List<CourseOfStudy> convert(List<org.niewidoczniakademicy.rezerwacje.core.model.csv.CourseOfStudy> cos)
```

Comment:
Same here (and also below, just refactor all in this file). Also I would go for courseOfStudy for CourseOfStudy class and courseOfStud**ies** for Lists to keep the code easy to read.